study.md

```diff
@@ -93,15 +95,15 @@ Ruby:
 ```ruby
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
-# your answer here
+array.map { |n| n * n }
 ```
 
 JavaScript:
 
 ```javascript
 const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
-// your answer here
+let squares = array.map(Math.sqrt);
```

Comment:
This will cause an error since `Math.sqrt` requires an argument.  Also, we want the squares of the numbers in the array, not the square roots.